uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/annual-entitlement-change-log.scss

```diff
@@ -0,0 +1,9 @@
+.annual-entitlement-change-log-component {
```

Comment:
Does it look OK on mobile?

Reply:
This screen can only be accessed from the admin portal, so the lack of zoom=1 doesn't help at all.

![download](https://user-images.githubusercontent.com/1642119/32511499-6607496a-c3a0-11e7-9449-a79032e8db26.png)

Regardless, I have made the table adapt to the modal container by adding a `.table-responsive` to it. This will help when the admin supports mobile, but also it helps right now when the entitlements are too many to display in one screen:

![download](https://user-images.githubusercontent.com/1642119/32511606-c30d19be-c3a0-11e7-9fb7-9c5c6eea33a4.png)
(screen below 400px width)

Side note: small screens (< 1200px) don't make the modal look any different from big screens.